docs/java-run-your-first-app.md

```diff
@@ -98,9 +94,7 @@ When the Worker starts it begins polling the Task Queue. The first Task it finds
 
 Congratulations, you have run a Temporal Workflow application!
 
-<img class="tutorial-section-icon" src={require('../static/img/docs/warning.png').default} />
-
-## Failure simulation
+## Failure simulation &nbsp;&nbsp;![](/img/docs/warning.png)
```

Comment:
![image](https://user-images.githubusercontent.com/6764957/94937517-41fb6500-0502-11eb-9369-cde1699dc996.png)

does this look overly alarming? (just asking, no strong opinion). may wish to just use emoji

Reply:
Yes

Reply:
I updated it to a yellow one. But I know what you mean - almost want one that shows strength. Unicorn? 
![unicorn](https://user-images.githubusercontent.com/34380806/95093038-8d219d80-06e5-11eb-8664-c199752a0b72.png)